### DISCUSSION TITLE
Cron trigger to change minReplicaCount

### DISCUSSION THREAD

**Discussion by Oscaarjs** [kedacore/keda#5708] [Q&A / Need Help] [answered]
Is there any way of letting a cron trigger setting the minReplicaCount on a ScaledObject?

My envisioned use-case is an object that during office hours can scale from 1-N replicas (to always have 1 replica available). Outside of office hours i'd like to let it scale down to 0 (that is, scale from 0-N instead). As far as I can see the cron trigger itself only takes in a "desiredReplicas". But is there a workaround for this?

**Comment by zroubalik** [comment, marked as answer]
Unfortunately this can be achieved with KEDA at the moment.
